### PR TITLE
Alpha fixes

### PR DESCRIPTION
**Pull request by k88hudson**
Just a couple of little touch-ups before we ship alpha

### CODE REVIEW

**Comment by k88hudson** [comment]
@rlr @oyiptong or @pdehaan R?

File: content-src/components/Search/Search.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ const Search = React.createClass({
     };
   },
   render() {
-    return (<div className="search-wrapper">
+    return (<form className="search-wrapper">
       <span className="search-label fa fa-search" />
-      <input ref="input" placeholder="Search" required />
+      <input ref="input" type="search" placeholder="Search" required />
```
Comment (pdehaan) [inline content-src/components/Search/Search.js]: I'm a bit confused here. Even though we specify `required`, I can leave the field blank and click Enter or press the arrow icon and I get a brief tooltip pop up saying "Please fill out this field." and then the form immediately submits to Yahoo!.

File: content-src/components/NewTabPage/NewTabPage.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ const NewTabPage = React.createClass({
           </section>
 
           <section>
-            <GroupedActivityFeed title="Top Activity" sites={props.History.rows} length={20} />
+            <GroupedActivityFeed title="Top Activity" sites={props.History.rows} length={6} />
```
Comment (pdehaan) [inline content-src/components/NewTabPage/NewTabPage.js]: Was this to match a UX spec or prevent v-scrolling or something, ~~or just a speed thing so we don't need to send as many URLs to embedly proxy~~ (nope, I'm still sending 11 `urls=` to embedly proxy, we're just only showing 6 of them)?

Comment (k88hudson) [inline content-src/components/NewTabPage/NewTabPage.js]: We use all the URLs on the Timeline page, just not on NewTab

File: content-src/components/Search/Search.js (hunk 1 above)
Comment (k88hudson) [inline content-src/components/Search/Search.js]: ah good call, ill add real disabling

**Comment by pdehaan** [comment]
Cool, thanks for the clarification.

R+ from me, with that one disabled form still submitting nit thing.

**Comment by k88hudson** [comment]
@pdehaan thanks!

**Pull request merged**